src/solver/conv_asm_implicit_gemm_gtc_fwd_nhwc.cpp

```diff
@@ -792,6 +792,11 @@ ConvAsmImplicitGemmGTCDynamicFwdXdlopsNHWC::GetWorkspaceSize(const ConvolutionCo
 
 bool ConvAsmImplicitGemmGTCDynamicFwdXdlopsNHWC::IsApplicable(const ConvolutionContext& ctx) const
 {
+#if WORKAROUND_ISSUE_1317
+    if(ctx.IsLayoutDefault())
```

Comment:
~since #1324 resolve this issue, this logic can be removed?~
sorry I saw https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1324#issuecomment-986093145, seems not resolved

Reply:
@atamazov are there other missing cases where we still get the same error? #1324 indeed has fixed the issue raised in the JIRA ticket I think.

Reply:
> 

OK, I'll check.

Reply:
@junliume  @shaojiewang 
> are there other missing cases where we still get the same error? 

Please see #1317. I am suspecting this is related to `n_groups > 1` && OCL BE.